### PR TITLE
API: add WebSocket

### PR DESCRIPTION
**Pull request by caugner**
Sources:
1. MDN compat data (legacy table + inline boxes):
   https://developer.mozilla.org/en-US/docs/Web/API/WebSocket
2. Firefox commit data:
   https://github.com/mozilla/gecko-dev/

### CODE REVIEW

File: api/WebSocket.json [hunk 1]
```diff
@@ -0,0 +1,732 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "WebSocket": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket",
```
Comment (connorshea) [inline api/WebSocket.json]: Incorrect URL

**Comment by connorshea** [comment]
This one is weird because the subfeatures don't actually have individual articles, and the article's format differs significantly from most other Web API pages on MDN. So I don't really know if this PR is correct or not :D

File: api/WebSocket.json (hunk 1 above)
Comment (caugner) [reply api/WebSocket.json]: Good catch as usual, fixed now in df5f7da.

**Comment by caugner** [comment]
You're right! Since most other pages are structured differently, I think it would be best to "refactor" the MDN article. I wouldn't mind doing that, though I don't currently have permissions to create pages on MDN.

@JeremiePat To create those new pages (`/docs/Web/API/WebSocket/*`), do you think it makes more sense to file a bug, or to ask for permission to create new pages? (e.g. [WebSocket/WebSocket](https://developer.mozilla.org/docs/Web/API/WebSocket/WebSocket)).

**Comment by JeremiePat** [comment]
@caugner Definitely ask for page creation permission ;)

**Comment by caugner** [comment]
@connorshea I have migrated the [WebSocket article](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) now, so we're good to go from my side.

File: api/WebSocket.json [hunk 2]
```diff
@@ -0,0 +1,732 @@
+{
+  "api": {
+    "WebSocket": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket",
+        "support": {
+          "webview_android": {
```
Comment (connorshea) [inline api/WebSocket.json]: Ideally `webview_android` would be at the bottom so this list is sorted alphabetically, but that's probably not really a blocker.

File: api/WebSocket.json [hunk 3]
```diff
@@ -0,0 +1,732 @@
...
+          "firefox": [
+            {
+              "version_added": "11",
+              "notes": "See <a href='https://bugzil.la/695635'>Bug 695635</a>."
+            },
+            {
+              "version_added": "7",
+              "version_removed": "11",
+              "prefix": "Moz",
+              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>Bug 711205</a>)."
```
Comment (connorshea) [inline api/WebSocket.json]: also bug should be lowercase or else it'll fail your lints :P #2511

**Comment by a2sheppy** [comment]
The documentation for WebSocket definitely needs a sizeable refactoring. It was an experimental format during a time of flux in our API documentation, unfortunately.

**Pull request merged**